control/controlclient/auto.go

```diff
@@ -168,39 +170,67 @@ func (c *Auto) Start() {
 func (c *Auto) sendNewMapRequest() {
 	c.mu.Lock()
 
-	// If we're not already streaming a netmap, or if we're already stuck
-	// in a lite update, then tear down everything and start a new stream
-	// (which starts by sending a new map request)
-	if !c.inPollNetMap || c.inLiteMapUpdate || !c.loggedIn {
+	// If we're not already streaming a netmap, then tear down everything
+	// and start a new stream (which starts by sending a new map request)
+	if !c.inPollNetMap || !c.loggedIn {
```

Comment:
reset `liteMapUpdateCancels` here too?

Reply:
I actually moved the `liteMapUpdateCancels` reset into `cancelMapSafely`, since I can't think of a reason not to reset it there and (IMO) it's less likely to be forgotten. Thoughts on that?

Reply:
yeah I was going to suggest something similar, sgtm